kgitb/github.py

```diff
@@ -12,34 +12,34 @@
 
 
 def protect(string):
-    return string.replace("[", r"\[")
+    return string.replace('[', r'\[')
 
 
 def quote(string):
-    return ">" + string.replace("\n", "\n>")
+    return '>' + string.replace('\n', '\n>')
 
 
 def get_commits(url):
-    commits = requests.get(url, params={"access_token": GH_TOKEN})
+    commits = requests.get(url, params={'access_token': GH_TOKEN})
     return commits.json()
 
 
 def get_comments(url):
-    comments = requests.get(url, params={"access_token": GH_TOKEN})
+    comments = requests.get(url, params={'access_token': GH_TOKEN})
     result = []
     for comment in comments.json():
-        if comment["body"].startswith(HIDDEN):
+        if comment['body'].startswith(HIDDEN):
             result.append(comment)
     return result
 
 
 def delete_comment(comment):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)